content/en/02-installing-pixie/index.md

```diff
@@ -16,4 +16,5 @@ In this section we'll cover steps to install Pixie in specific K8s environments:
 - [Install Guides](/installing-pixie/install-guides)
     - [Community Cloud for Pixie](/installing-pixie/install-guides/community-cloud-for-pixie)
     - [Self-Hosted Pixie](/installing-pixie/install-guides/self-hosted-pixie)
+    - [Production Readiness - Shared Cloud](/installing-pixie/install-guides/production-readiness-shared-cloud)
```

Comment:
Can we also add a link to this page in the "Self-hosted Pixie" page? At the bottom there is the "Production Readiness (advanced) section", we should link out to this page there.

Reply:
Updated both Self-Hosted Pixie and Air-gapped installation pages and added a link to this page. Fixed through https://github.com/pixie-io/docs.px.dev/pull/245/commits/eb0693c49c18d9f219c0275b718d4f3d82dbbf5a